content/articles/index/index.mdx

```diff
@@ -0,0 +1,38 @@
+---
+subTitle: "# SSW Articles\n\nSSW has been developing software since 1990. Below are some of our great articles over the years. You can see much more specialized content in every area of software development on\_SSW Rules.\n"
+sidebar: |
+  <MicrosoftPanel />
+seo:
+  title: 'SSW Articles - Sydney, Melbourne, Brisbane''s Premier Software Consultants'
+  description: "SSW has been developing software since 1990. Below are some of our great articles over the years. You can see much more specialized content in every area of software development on\_SSW Rules."
+title: SSW Articles
+showMicroservices: true
+microservicesTitle: 2-Day Pre-Migration Assessment Engagement
+microservicesDescription: >-
+  Get a solid foundation for your .NET 8 migration project, ensuring you are
+  well-prepared to tackle the migration with confidence.
+microservicesUrl: /
```

Comment:
I think that this should this be an object instead of a few props related by a naming convention

Maybe call it sidebar?

Reply:
done

Reply:
This still looks like multiple props instead of an object
Also still called microservices… was code not pushed?